### PR TITLE
base.html: Add script and style blocks

### PR DESCRIPTION
**Pull request by NikolaJankovic**
issue #87

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -17,6 +17,8 @@
     <link rel = 'stylesheet' href="{% static 'home/base.css' %}">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet"/>
+    <style></style>
```
Comment (sagar-kalra) [inline templates/base.html]: also add template inheritance blocks to both.

**Comment by sagar-kalra** [comment]
Improve commit message, see our readme for guidelines and squash your commits, then you are good to go.

**Comment by NikolaJankovic** [comment]
The readme doesn't detail any specifications regarding commit messages, it just mentions that they should be good. From the other commits the only difference I can see is that some note the specific file that's been updated; is that the only required change?

File: templates/base.html [hunk 2]
```diff
@@ -17,8 +17,14 @@
     <link rel = 'stylesheet' href="{% static 'home/base.css' %}">
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet"/>
-    <style></style>
-    <script></script>
+    <style>
```
Comment (gurpreetsingh00885) [inline templates/base.html]: The tags are not needed. They will be added by the one who includes the extra css or js.
Remove them.

**Comment by divyam3897** [comment]
@NikolaJankovic are you still working on it?

**Comment by divyam3897** [comment]
It has been fixed in https://github.com/pclubuiet/website/commit/e41662c0496ce5439cead0c88d63474ca77c265f Thank you for your PR.

**Pull request closed without merging**